tests/test_pillow_simd.py

```diff
@@ -1,7 +1,13 @@
 # Copyright 2021 MosaicML. All Rights Reserved.
 
+import os
+import platform
+
 import PIL
+import pytest
 
 
+@pytest.mark.skipif('composer-python' not in os.environ["PATH"] or 'Linux' not in platform.system(),
```

Comment:
Hmm, could we use a different check than `'composer-python' not in os.environ["PATH"]` to determine if we're in the composer docker? This test should not be linked to the python path of the dockerfile.

Instead, would it work if we checked to see if we were running in a CI environment?
e.g. `if os.environ.get("CI") != "true"`  See: https://ci.int.mosaicml.com/env-vars.html/
Granted one may be running the docker image outside of CI/CD, in which case this test would not run, but I think that would be fine since it would still run for our builds?

I would also be OK with removing this test and instead having a test script to run when building the docker image (and putting a test like this in there). (OK with punting this change for now, until we have more jenkins cycles)